### PR TITLE
Update Patch Files [Implementation, Language Family]

### PR DESCRIPTION
**Pull request by joshuatcasey**
Update Patch Files [Implementation, Language Family]

### CODE REVIEW

**Comment by ryanmoran** [comment]
I can't think of anything we should really be that concerned about.

**Comment by sophiewigmore** [comment]
@joshuatcasey @ryanmoran context on why `build.sh` was omitted: https://github.com/paketo-buildpacks/github-config/pull/360#issuecomment-953902335

**Comment by joshuatcasey** [comment]
> @joshuatcasey @ryanmoran context on why `build.sh` was omitted: [#360 (comment)](https://github.com/paketo-buildpacks/github-config/pull/360#issuecomment-953902335)
> 
> > With time and evidence these may be added, but for now they will not be auto-labeled as patch changes.

I recommend we add them - do we have any evidence not to?

**Comment by sophiewigmore** [comment]
It rarely changes so there's not really any benefit to having it or not having it. I don't think we have any evidence either way because it hasn't changed. I'm fine with leaving it off unless we see a situation in which it would be helpful, but that's just me.

File: implementation/.github/.patch_files [hunk 1]
```diff
@@ -19,11 +19,14 @@
...
 README.md
+go.mod
+go.sum
+scripts/.util/builders.sh
+scripts/.util/git.sh
 scripts/.util/print.sh
 scripts/.util/tools.json
 scripts/.util/tools.sh
-scripts/.util/builders.sh
+scripts/build.sh
```
Comment (fg-j) [inline implementation/.github/.patch_files:29]:
```suggestion
```

Comment (joshuatcasey) [reply implementation/.github/.patch_files:29]: will commit separately

**Comment by joshuatcasey** [comment]
Updated to remove `scripts/build.sh` from the implementation patch files.

File: implementation/.github/.patch_files [hunk 2]
```diff
@@ -4,13 +4,13 @@
 .github/dependabot.yml
 .github/labels.yml
 .github/workflows/approve-bot-pr.yml
-.github/workflows/auto-merge.yml
-.github/workflows/check-pr-labels.yml
```
Comment (sophiewigmore) [inline implementation/.github/.patch_files:8]: why are these getting removed

**Comment by sophiewigmore** [comment]
@joshuatcasey you removed `build.sh` but added `create-release`, `push-buildpackage`, and `package.sh` which were all originally omitted for the same initial reason as `build.sh`. I'm just a little confused - why removed `build.sh` but leave those? Is there different reasoning for them?

**Comment by joshuatcasey** [comment]
> @joshuatcasey you removed `build.sh` but added `create-release`, `push-buildpackage`, and `package.sh` which were all originally omitted for the same initial reason as `build.sh`. I'm just a little confused - why removed `build.sh` but leave those? Is there different reasoning for them?

@sophiewigmore No different reasoning, just didn't explore the original issue to see why these were omitted. They have been removed now.

**Review by sophiewigmore — APPROVED** [review]

**Pull request merged**